src/App/Fossa/Container.hs

```diff
@@ -47,7 +47,7 @@ import Path.IO (getCurrentDir)
 newtype ImageText = ImageText {unImageText :: Text} deriving (Show, Eq, Ord)
 
 imageTextArg :: Parser ImageText
-imageTextArg = ImageText . pack <$> argument str (metavar "IMAGE" <> help "The image to scan")
+imageTextArg = ImageText . toText @String <$> argument str (metavar "IMAGE" <> help "The image to scan")
```

Comment:
I think you should be able to remove the `String`-handling part of this entirely:

```suggestion
imageTextArg = ImageText <$> argument str (metavar "IMAGE" <> help "The image to scan")
```

Reply:
Generally, using string conversion with type application should NEVER need to happen.  Usually, one or the other is enough, and when it's not, there's almost certainly a hidden problem elsewhere causing this.  The original `pack` was unneeded as well.